gbpservice/nfp/configurator/agents/loadbalancer_v2.py

```diff
@@ -0,0 +1,978 @@
+
+        """
+        arg_dict = {'context': context,
+                    'pool': pool,
+                    }
+        # For Mitaka, use binding_key=pool['listeners'][0]['id']
+        # For Liberty, use binding_key=pool['listener']['id']
+        self._send_event(lb_constants.EVENT_DELETE_POOL, arg_dict,
+                         serialize=True,
+                         binding_key=pool['loadbalancer_id'],
+                         key=pool['id'])
+
+    def create_member(self, context, member):
+        """Enqueues event for worker to process create member request.
+
+        :param context: RPC context
+        :param member: member resource to be created
+
+        Returns: None
+
+        """
+        arg_dict = {'context': context,
+                    'member': member,
+                    }
+        self._send_event(lb_constants.EVENT_CREATE_MEMBER, arg_dict,
+                         serialize=True,
+                         binding_key=member['pool']['loadbalancer_id'],
+                         key=member['id'])
+
+    def update_member(self, context, old_member, member):
+        """Enqueues event for worker to process update member request.
+
+        :param context: RPC context
+        :param old_member: old member resource to be updated
+        :param member: new member resource
+
+        Returns: None
+
+        """
+        arg_dict = {'context': context,
+                    'old_member': old_member,
+                    'member': member,
+                    }
+        self._send_event(lb_constants.EVENT_UPDATE_MEMBER, arg_dict,
+                         serialize=True,
+                         binding_key=member['pool']['loadbalancer_id'],
+                         key=member['id'])
+
+    def delete_member(self, context, member):
+        """Enqueues event for worker to process delete member request.
+
+        :param context: RPC context
+        :param member: member resource to be deleted
+
+        Returns: None
+
+        """
+        arg_dict = {'context': context,
+                    'member': member,
+                    }
+        self._send_event(lb_constants.EVENT_DELETE_MEMBER, arg_dict,
+                         serialize=True,
+                         binding_key=member['pool']['loadbalancer_id'],
+                         key=member['id'])
+
+    def create_healthmonitor(self, context, healthmonitor):
+        """Enqueues event for worker to process create health monitor request.
+
+        :param context: RPC context
+        :param health_monitor: health_monitor resource to be created
+        :param pool_id: pool_id to which health monitor is associated
+
+        Returns: None
+
+        """
+        arg_dict = {'context': context,
+                    'health_monitor': healthmonitor
+                    }
+        self._send_event(lb_constants.EVENT_CREATE_HEALTH_MONITOR,
+                         arg_dict, serialize=True,
+                         binding_key=healthmonitor['pool']['id'],
```

Comment:
binding_key should be loadbalancer_id, right ? Same comment for update_healthmonitor and delete_healthmonitor